microsoftgraph/microsoft-graph-explorer-v4
# Feature: download api manifest

## Overview

Shows what to do with an API Manifest when you download it
Closes #2446 
Closes #2409
Closes #2687 

## Testing Instructions

* Add items to a collection
* Preview collection
* Click the API Manifest button
* Hit download and see a manifest downloaded to your device.

## Notes

- [ ] Get the correct manifest instructions from @RabebOthmani 
- [ ] Ensure items are localised
